radixdlt/babylon-wallet-ios
# [ABW-2527] Ditch Hardcoded Strings

Jira ticket: ABW-2527

## Description
We still have a large number of hardcoded strings in the iOS repo, this PR gets rid of them.

A number of strings have incorrect or unsuitable keys. These two PRs adds new keys for these, without removing the old ones - that will be done in the next PR.

A corresponding PR to the localization repo can be found [here](https://github.com/radixdlt/apps-localization-src/pull/220). It should be merged first, then the automatic PR to the wallet repo should be merge, and finally the TEMP commit in this PR should be dropped before merging this PR.

### Notes
The stings related to MFA have not been migrated, since they are (presumably) subject to change.

Additionally, we show a message "Seed phrase imported" on the HUD when entering a seed phrase.

Also changes the title for Transaction Review to the original "Review **your** transaction", which necessitated some changes to the jagged receipt view!

<img src="https://github.com/radixdlt/babylon-wallet-ios/assets/123396602/607f9c22-3c05-4255-932c-123c892c87c0" width="200" />

## How to test
Check that all strings look correct, especially the ones added or changed (see this PR or the other one).

Also test that the "Seed phrase imported" message shows up as appropriate
